pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    pub: Tests that have to be run on images that are already published into cloud providers' marketplaces.
```

Comment:
Apostrophe typo

Reply:
Ad discussed via Slack, this is also valid grammatically speaking.